apps/explorer/src/components/activity/Activity.tsx

```diff
@@ -63,7 +62,7 @@ export function Activity({ initialLimit, disablePagination }: ActivityProps): JS
                 id: AUTO_REFRESH_ID,
             });
         } else {
-            toast.success('Auto-refresh paused', { id: AUTO_REFRESH_ID });
+            toast.warning('Auto-refresh paused', { id: AUTO_REFRESH_ID });
```

Comment:
both these toasts should be info imo, is a play/pause feature, success and warning are confusing imo